requirements.txt

```diff
@@ -2,3 +2,4 @@ anymarkup
 jsonpointer
 requests
 websocket-client
+GitPython
```

Comment:
I agree @dustymabe in regards to the extra dependency. This call will be rarely used. Only used if the person wants to create their own "nulecule library" hosted on a different docker container, etc.

Do you agree that we should instead take the `generation` command and add it to `projectatomic/nulecule-library` repo?